openmls/src/group/errors.rs

```diff
@@ -185,17 +187,35 @@ pub enum StageCommitError {
     UpdatePathError(#[from] ApplyUpdatePathError),
 }
 
-// === Crate errors ===
-
 /// Create commit error
 #[derive(Error, Debug, PartialEq, Clone)]
 pub enum CreateCommitError {
     #[error(transparent)]
     LibraryError(#[from] LibraryError),
+    #[error("Missing own key to apply proposal.")]
+    OwnKeyNotFound,
     #[error("The Commit tried to remove self from the group. This is not possible.")]
     CannotRemoveSelf,
+    #[error("The proposal queue is missing a proposal for the commit.")]
+    MissingProposal,
+    #[error("A proposal has the wrong sender type.")]
+    WrongProposalSender,
```

Comment:
The name indicates that something about the sender is wrong but the comment says sender type. That's confusing. The error could be read as the sender is not the expected sender or something else. Maybe `WrongProposalSenderType`?